Podfile.lock

```diff
@@ -1,83 +1,101 @@
 PODS:
-  - Firebase/Analytics (7.5.0):
+  - Firebase/Analytics (8.3.0):
```

Comment:
this updates seem kind of random, we shouldn't update libs randomly without a proper retest